packages/extract-react-types/src/index.js

```diff
@@ -1392,6 +1393,18 @@ function exportedComponents(programPath, componentsToFind: 'all' | 'default', co
         return;
       }
 
+      if (isMemo && genericTypeParams.node) {
+        const component = convertReactComponentFunction(
+          genericTypeParams,
+          context,
+          genericTypeParams.get('params.0')
```

Comment:
Can you explain why this is `'params.0'`?

Reply:
So props are the is the first generic in the list `params.0`. Where in `forwardRef`, props are the second `forwardRef<Element, Props>`